dashboard/app/controllers/api/v1/test_logs_controller.rb

```diff
@@ -10,9 +10,10 @@ def get_logs_since
     boundary_time = Time.at(params[:time].to_i)
     bucket = Aws::S3::Bucket.new('cucumber-logs')
     objects = bucket.objects({prefix: "#{params[:prefix]}/"})
-    render json: objects.select {|summary|
+    json_result = render json: objects.select {|summary|
       boundary_time <= summary.last_modified
-    }.map {|summary|
+    }
+    json_result.map {|summary|
       {
         key: summary.key,
         last_modified: summary.last_modified
```

Comment:
That doesn't seem right. The rendered json is supposed to be the result of the map. Here it's rendering the pre-mapped results. I think it should be something like:

```ruby
selected_objects = objects.select{|summary| boundary_time <= summary.last_modified}
json_result = selected_objects.map do |summary|
  {
      key: summary.key,
      last_modified: summary.last_modified
  }
end
render json: json_result
```

This is also easier to read since render is the last line.